cmake/external/tvm.cmake

```diff
@@ -4,13 +4,19 @@ if (onnxruntime_USE_TVM)
   FetchContent_Declare(
     tvm
     GIT_REPOSITORY https://github.com/apache/tvm.git
-    GIT_TAG        bc492acd7677dd7875b14f9ee46beef658955441
+    GIT_TAG        3425ed846308a456f98404c79f6df1693bed6377
   )
 
   FetchContent_GetProperties(tvm)
   if(NOT tvm_POPULATED)
     FetchContent_Populate(tvm)
-    file(CREATE_LINK ${tvm_BINARY_DIR} ${tvm_SOURCE_DIR}/build SYMBOLIC)
+    if (WIN32)
+      execute_process(
+        COMMAND ${CMAKE_COMMAND} -E create_symlink ${tvm_BINARY_DIR}/Release ${tvm_SOURCE_DIR}/build
```

Comment:
Use `${tvm_BINARY_DIR}/${CMAKE_BUILD_TYPE}` instead `${tvm_BINARY_DIR}/Release` to automatically detect the build type of the project.